.github/docker-images/al2-x64/Dockerfile

```diff
@@ -21,15 +20,13 @@ RUN yum -y update \
     which \
     && yum clean all \
     && rm -rf /var/cache/yum \
-    && ln -s /usr/bin/cmake3 /usr/bin/cmake \
-    && ln -s /usr/bin/ctest3 /usr/bin/ctest \
```

Comment:
trivial: Why are we removing this? I don't see any error regarding this in build logs.

Reply:
al2 cmake install already creates soft link, so it errors out when you try to create them again. there should be an old generation run where this error occured